properties.py

```diff
@@ -1460,14 +1465,14 @@ class RendermanObjectSettings(bpy.types.PropertyGroup):
         default=False)
     motion_segments = IntProperty(
         name="Motion Samples",
-        description="Number of motion samples to take for multi-segment motion blur",
+        description="Number of motion samples to take for multi-segment motion blur.\nUse higher numbers for curved motion trails.",
```

Comment:
mmmmmm.... curved motion tails?  Maybe "Use higher numbers if motion blur is segmented?